eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -25,18 +25,6 @@ steps:
       inlineScript: |
         Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
-  - task: MicroBuildLocalizationPlugin@1
-    displayName: "Install Localization Plugin"
-
-  - task: MicroBuildSigningPlugin@1
-    inputs:
-      signType: "$(SigningType)"
-      esrpSigning: "true"
-    displayName: "Install Signing Plugin"
-
-  - task: MicroBuildSwixPlugin@1
```

Comment:
the build failing due to missing this plugin

Reply:
The CI logs showed the plugin was getting installed, but getting a binlog, I could see it wasn't being imported for some reason. But the plugin install logs say to make sure to use the MicroBuild core package version 1.0.0, so I upgraded it and it started working 🤷